schemas/rum/resource-schema.json

```diff
@@ -242,6 +242,13 @@
               "description": "trace identifier in decimal format",
               "pattern": "^[0-9]+$",
               "readOnly": true
+            },
+            "rule_psr": {
+              "type": "number",
+              "description": "sample rate in integer format",
```

Comment:
Maybe we need to be more precise as to which sampling rate this refer to. We have (at least) three: RUM session, RUM premium session and Tracing